src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -35,6 +40,233 @@ public ProjectView()
             _versions.ItemContainerStyle = style;
         }
 
+        private TextBox _textBox;
+
+        private TextBox TextBox
+        {
+            get
+            {
+                if (_textBox == null) return _versions.Template.FindName("PART_EditableTextBox", _versions) as TextBox;
+                return _textBox;
```

Comment:
Can we flip this around so that it's `return null` when not finding the editable textbox? 
Is it ever normal for it not to find that textbox?

Reply:
Im not sure I understand your comment, here Im trying to save the `TextBox` in a variable to avoid recalculations later on the code. Or will `_versions.Template.FindName("PART_EditableTextBox", _versions) as TextBox` be stored in memory and I can delete this object?

Reply:
I didn't see that the property was necessary. I think you can just use the `_textBox` field. 
Why not just: `_textBox = _versions.Template.FindName("PART_EditableTextBox", _versions) as TextBox` ?

Restating my question: does `FindName` ever return a `null`?
If it does, there's code referencing `TextBox` that will break.

Reply:
The only time it is null is when the component loads, that why I dont just use
`_textBox = _versions.Template.FindName("PART_EditableTextBox", _versions) as TextBox`